paddle/fluid/distributed/store/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ cc_library(
   SRCS tcp_store.cc tcp_utils.cc socket.cpp
   DEPS enforce glog)
 
+if(NOT WIN32)
```

Comment:
It is not professional to cover (ignore) errors, but it may be temporary. You just have to reckon with the fact that problems may pile up in the future.

Reply:
@pmajchrzak will you please try this PR in your environment, if build error disappears, I will request review.

Reply:
I tried to do something like that: 
```
curl https://github.com/Aganlengzi/Paddle/commit/8a0a47b2e2681f310d46da1a665f503fc798789a.diff | git apply -v 
```
but get:
```
+ cd /repos/paddle_paddle
+ curl https://github.com/Aganlengzi/Paddle/commit/8a0a47b2e2681f310d46da1a665f503fc798789a.diff
+ git apply -v
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   653  100   653    0     0   3910      0 --:--:-- --:--:-- --:--:--  3910
Checking patch paddle/fluid/distributed/store/CMakeLists.txt...
error: while searching for:
  SRCS tcp_store.cc tcp_utils.cc socket.cpp
  DEPS enforce glog)

set_source_files_properties(tcp_store.cc PROPERTIES COMPILE_FLAGS
                                                    "-Wno-error=unused-result")

if(NOT WIN32)
  cc_test(

error: patch failed: paddle/fluid/distributed/store/CMakeLists.txt:3
error: paddle/fluid/distributed/store/CMakeLists.txt: patch does not apply
```

Reply:
@pmajchrzak hi, this PR contains two commits, maybe you should apply them both:
https://github.com/Aganlengzi/Paddle/commit/41f70afd240f58009434592326c5caae7407931a.diff
https://github.com/Aganlengzi/Paddle/commit/8a0a47b2e2681f310d46da1a665f503fc798789a.diff